web/src/app/components/PageLayout/Header.tsx

```diff
@@ -0,0 +1,36 @@
+
+export default function Header() {
+    return (
+        <div className="flex w-full flex-col items-center space-y-2 pt-12 sm:flex-row sm:space-y-0 sm:space-x-6 bg-gray-200">
```

Comment:
These are just pure lines of code for icons. Can change them later when we install a icon pack. I didn't want to to install a pack so I just got the raw code for them.

Reply:
![image](https://github.com/jonathanplacatka/typing-game/assets/93963545/0930571e-80cf-4e3b-ad00-46a4a27e1dab)